protocol/dubbo3/dubbo3_protocol.go

```diff
@@ -242,16 +242,16 @@ func (dp *DubboProtocol) openServer(url *common.URL, tripleCodecType tripleConst
 		}
 	}
 
-	if maxCall := url.GetParam(constant.MaxServerRecvMsgSize, ""); maxCall != "" {
-		if size, err := strconv.Atoi(maxCall); err == nil && size != 0 {
-			opts = append(opts, triConfig.WithGRPCMaxServerRecvMessageSize(size))
-		}
+	maxServerRecvMsgSize := constant.DefaultMaxServerRecvMsgSize
+	if recvMsgSize, err := humanize.ParseBytes(url.GetParam(constant.MaxServerRecvMsgSize, "")); err == nil && recvMsgSize != 0 {
```

Comment:
Plus, `recvMsgSize` and `sendMsgSize` should be greater than 0. So the code should be:

```suggestion
	if recvMsgSize, err := humanize.ParseBytes(url.GetParam(constant.MaxServerRecvMsgSize, "")); err == nil && recvMsgSize > 0 {
	// do something...
}
```